overlord/snapshotstate/backend/helpers.go

```diff
@@ -152,26 +153,53 @@ func allUsers() ([]*user.User, error) {
 	return users, nil
 }
 
+var (
+	sysGeteuid   = sys.Geteuid
+	execLookPath = exec.LookPath
+)
+
+func pickUserWrapper() string {
+	// runuser and sudo happen to work the same way in this case.
+	// The main reason to prefer runuser over sudo is that runuser
+	// is part of util-linux, which is considered essential,
+	// whereas sudo is an addon which could be removed.
+	for _, cmd := range []string{"runuser", "sudo"} {
+		if lp, err := execLookPath(cmd); err == nil {
+			return lp
+		}
+	}
+	return ""
+}
+
+var sudo = pickUserWrapper()
```

Comment:
I think the variable name is no longer appropriate. Perhaps `launcher`?

Reply:
I could unify things:
* `pickUserWrapper` is alright, i think
* so `sudo` -> `userWrapper`
* so `maybeRunuserCommand` -> `userWrapCommand` ?

Reply:
`runAsUser`